src/core/root.js

```diff
@@ -39,6 +43,25 @@ export async function makeContext(
   } = opts
   const log = makeLog(onLog, 'edge-core')
 
+  function onLog(event: EdgeLogEvent) {
+    // $FlowFixMe
+    const { logSettings = {} } = redux.getState() ? redux.getState() : {}
```

Comment:
Change to `const logSettings = redux != null ? redux.getState().logSettings : { sources:[], logLevel: 'warn' }`